raspi-config

```diff
@@ -1454,6 +1455,7 @@ do_internationalisation_menu() {
   FUN=$(whiptail --title "Raspberry Pi Software Configuration Tool (raspi-config)" --menu "Localisation Options" $WT_HEIGHT $WT_WIDTH $WT_MENU_HEIGHT --cancel-button Back --ok-button Select \
     "I1 Change Locale" "Set up language and regional settings to match your location" \
     "I2 Change Timezone" "Set up timezone to match your location" \
+    "I3 Change Keyboard Layout" "Set the keyboard layout to match your keyboard" \
```

Comment:
You can take out the whole `if` block and is_ssh command, right?
Sorry to be nit-picky.

Reply:
Yup. Just pushed the code I was using for testing and didn't clean it. Lazy.
fixed